src/objects/npe03__Recurring_Donation__c.object

```diff
@@ -259,6 +259,16 @@
                 </value>
             </valueSetDefinition>
         </valueSet>
+    </fields>
+        <fields>
+        <fullName>EndDate__c</fullName>
+        <description>After this date, NPSP no longer creates installments for this Recurring Donation and updates Status to Closed.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>After this date, NPSP no longer creates installments for this Recurring Donation and updates Status to Closed.</inlineHelpText>
```

Comment:
@sam-knox Does this apply to Open-Ended RD's only? If so, do you think that needs to be called out in the help text/description?

Reply:
@force2b - I actually don't know. I've been wondering how this will work with Fixed length. I'll post a message on the WI.